src/pudl/helpers.py

```diff
@@ -1792,7 +1792,7 @@ def fix_boolean_columns(
     """
     fillna_cols = {col: pd.NA for col in boolean_columns_to_fix}
     boolean_replace_cols = {
-        col: {"Y": True, "N": False, "X": False, "U": pd.NA}
+        col: {"Y": True, "N": False, "X": False, "U": pd.NA, 1: True, 0: False}
```

Comment:
Huh! First time we've had to replace 0's and 1's with their boolean counterparts 🎉 

I'm slightly surprised that pandas doesn't coerce these automatically - one of my favorite Weird Python Things is that `True` and `False` are literally the integers `1` and `0`... but I guess that makes sense.

```
>>> isinstance(True, int)
True
>>> True + True
2
>>> (True + True) * 5
10
>>> 1 == True
True
>>> 0 == False
True
```

Reply:
oh you are totally right here. I'll remove this. I forgot that I haven't made the schema for these tables yet and therefor can't yet apply the schema dtypes and so of course I am not yet getting the proper bool outputs.